packages/training/book/src/appendix/env-vars.md

```
@@ -0,0 +1,287 @@
+
+Create `.env` in project root:
+
+```bash
+# .env
+DATABASE_URL=postgresql://babylon:password@localhost:5432/babylon
+WANDB_API_KEY=your_key
+OPENAI_API_KEY=sk-your-key
+```
+
+Python auto-loads via `dotenv`:
+```python
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)